src/metrics.cpp

```diff
@@ -108,34 +109,22 @@ double GetLocalSolPS()
     return miningTimer.rate(solutionTargetChecks);
 }
 
-int EstimateNetHeightInner(int height, int64_t tipmediantime,
-                           int heightLastCheckpoint, int64_t timeLastCheckpoint,
-                           int64_t genesisTime, int64_t targetSpacing)
+int EstimateNetHeight(const Consensus::Params& params, int currentBlockHeight, int64_t currentBlockTime)
```

Comment:
This should be passed the `pindexBestHeader` height and time (or maybe just pass it a `const &CBlockIndex`, which will normally be `*pindexBestHeader`).